tutorials/zkml.md

```diff
@@ -0,0 +1,255 @@
+# zkML Rollup Tutorial with Sindri
+
+## Introduction - Code Once, Verify Anywhere
+
+This guide will show you how to build verifiable machine learning inference into your Rollkit rollup using zero-knowledge proofs generated on [Sindri](https://sindri.app).
+The rollup will rely on a local Celestia devnet for consensus and ensuring data availability.
+
+We will focus on using a ZK circuit that incorporates a compact deep neural network model to enable verifiable ML inference.
+In this guide, we will deploy the circuit on Sindri, obtain a smart contract verifier (automatically generated by Sindri) for that circuit, deploy it on Rollkit, create a proof of ML inference on Sindri, and verify the proof on-chain.
+
+This approach not only secures the verification process of machine learning models, but also leverages the decentralized security and scalability of Celestia's architecture.
+
+## Prerequisites
+
+- An operational Polaris EVM using Rollkit (see next step)
```

Comment:
Let's link to the polaris EVM tutorial here. It currently has a disclaimer so would be good to link!